### ISSUE TITLE
[4.0] com-messages broken

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#21090]
The links in com_messages are all wrong

Make sure that you have a message in your inbox.

Try and select that message by checking the box and you will be redirected to
index.php?option=com_fields&context=com_users.user

try and select all messages by checking the box in the table header and you will be redirected to
index.php?option=com_users&view=levels

Note also that the redirect is on click but these are checkboxes not submits

**Comment by SharkyKZ** [comment]
Template issue. Collapsed menu items aren't being hidden.
![untitled](https://user-images.githubusercontent.com/7325021/42688657-e15233a0-86a5-11e8-86df-afae01472316.png)
@C-Lodder @ciar4n can you take a look at this please.

**Comment by C-Lodder** [comment]
@brianteeman @SharkyKZ Not tested the new template so can't confirm, but try adding:

`overflow: hidden;`

to this line: https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/scss/blocks/_sidebar-nav.scss#L4

Or find it in the CSS file if you're not set up to compile SCSS

May be the wrong place though as there seem to be 2 SCSS files for the sidebar and I don't know if the one mentioned above it for something else or not.

**Comment by SharkyKZ** [comment]
That menu doesn't user `.sidebar-nav` class. I think this could be it:

```
.main-nav .collapse-level-1:not(.show) {
    display: block;
}
```
`.show` is not used also . Instead `.open` is added/removed from parent. So this seems to work:

```
.main-nav .open .collapse-level-1 {
   display: block;
}
```

**Comment by brianteeman** [comment]
good spot. - i didnt even think that could be the issue - its likely to effect other components then as well

**Comment by roland-d** [comment]
This issue drove me crazy yesterday. If you go to the Plugins list, try to drag a plugin, you can't. It will drag a submenu item and open that page when you let go.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21090">issues.joomla.org/tracker/joomla-cms/21090</a>.</sub>

**Comment by brianteeman** [comment]
I had a try at fixing it but my css foo is not strong enough

**Comment by brianteeman** [comment]
@ciar4n could you take a look at this please.

**Comment by ghost** [comment]
closed as having Pull Request #21316
